src/test/resources/config/application.yml

```diff
@@ -1,3 +1,8 @@
 spring:
     dataSource:
-        url: jdbc:hsqldb:mem:dddsample_test
\ No newline at end of file
+        url: jdbc:hsqldb:mem:dddsample_test
+    main:
+        allow-bean-definition-overriding: true
```

Comment:
What is this for?

Reply:
During spring boot2 it allows we define a bean with the same name,
and previous we define the bean with the same name for testing .
but when we upgrade to spring boot 3, there will be an error for it.
So I add this config for testing and  I need to tidy the test codes further.

Reply:
Got it!